cypress/integration/basic-functionalities/local-tests/reporter/Delete_report.spec.js

```diff
@@ -13,20 +13,27 @@ describe('Test for deleting a report', () => {
   context('Test for deleting a report', () => {
     it('should be able to delete a report successfully', () => {
       // We click on the Edit button on the created report
-      cy.get('table > tbody > :first-child() > :last-child() > a:nth-child(2)').click() 
```

Comment:
Can you, please, add a comment that you'll be deleting multiple/the rest of your reports. Got a bit confused here. Thanks